.gitignore

```diff
@@ -128,9 +128,8 @@ server/webapp/WEB-INF/rails.new/log/
 server/webapp/WEB-INF/rails.new/logs/
 server/webapp/WEB-INF/rails.new/node_modules/
 server/webapp/WEB-INF/rails.new/public/assets/
-server/webapp/WEB-INF/rails.new/vendor/bundle/jruby/1.9/doc/
-server/webapp/WEB-INF/rails.new/vendor/bundle/jruby/1.9/extensions/
 server/webapp/WEB-INF/rails.new/webpack/gen/
+server/webapp/WEB-INF/rails.new/vendor/
```

Comment:
Even if they were all from it, I wonder if we've had to make changes in some cases. :(

Also, with a change such as this, it's possible that a `git clone` and then a build will probably fail. Not immediately of course, since git won't remove ignored files, but if any new file gets added to `vendor`, it won't get checked in.